db/migrate/20150317210343_convert_target_minutes_to_hours.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertTargetMinutesToHours < ActiveRecord::Migration
+  def change
```

Comment:
Please add a temporary target class to the migration to be more robust against renaming/changing the model.
(like we did it in previous migrations
https://github.com/openSUSE/osem/blob/master/db/migrate/20141128073306_migrate_data_remove_column_include_cfp_in_splash_add_column_include_cfp.rb#L6)
See http://railsguides.net/change-data-in-migrations-like-a-boss/ for details!